.travis.yml

```diff
@@ -1,5 +1,13 @@
 sudo:       required
+language:   generic
 services:   docker
+
+tex-config:
```

Comment:
You could just put it below `env` and make them environmental variables!
This way `execute_tests.sh` doesn't have to parse them, they'll just be available.